src/services/offer.js

```diff
@@ -155,6 +156,35 @@ class OfferService {
         return offer;
     }
 
+    blockByCompany(owner) {
+        return Offer.updateMany(
+            { owner, isHidden: false },
+            {
+                isHidden: true,
+                hiddenReason: HiddenOfferReasons.COMPANY_BLOCKED,
+            },
+            { new: true },
```

Comment:
What does the this line do here? I tried searching about i but didn't find much

Reply:
In some updates it would return the new item (findAndUpdate for example), but in this case it just returns information on what was changed, so I guess this isn't needed